CHANGELOG.md

```diff
@@ -34,7 +34,7 @@ _None._
 
 ### Breaking Changes
 
-_None._
+- `nvm_install` utility is removed. See [MIGRATION.md](./MIGRATION.md#from-200-to-300) for details.
```

Comment:
I'm not an native English speaker, but I don't know why, the "is removed" wording sounds odd to me? 🤷 
```suggestion
- `nvm_install` utility has been removed. See [MIGRATION.md](./MIGRATION.md#from-200-to-300) for details.
```

Reply:
Thanks for the suggestions. Addressed in https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/73/commits/679f919cb64e8a8ec02aadb442d88c3c96104c51